src/@next/components/atoms/AddNewTile/test.tsx

```diff
@@ -0,0 +1,14 @@
+import { mount } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { AddNewTile } from ".";
+
+describe("<AddNewTile />", () => {
+  it("AddNewTile component exists and proper text content is passed to Tile component", () => {
```

Comment:
In my opinion it("should exists......") is better

Reply:
Fixed